include/boost/geometry/strategies/cartesian/mirror_box.hpp

```diff
@@ -0,0 +1,48 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2018 Oracle and/or its affiliates.
+// Contributed and/or modified by Vissarion Fisikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_CARTESIAN_MIRROR_BOX_HPP
+#define BOOST_GEOMETRY_STRATEGIES_CARTESIAN_MIRROR_BOX_HPP
+
+namespace boost { namespace geometry
+{
+
+
+namespace strategy { namespace mirror_box
+{
+
+struct cartesian
```

Comment:
What is this strategy? Is there mirror_box() algorithm? It doesn't do anything for cartesian. If there is no algorithm needing it and it doesn't do anything in castesian then my guess is that it's a utility used in some spherical and/or geographic strategy. If that's the case this shouldn't be a separate strategy.